tutorials/03_thermal_camera.md

```diff
@@ -152,6 +158,33 @@ Most of the code above is for the model - here's the key piece for temperature a
 
 Once the thermal system plugin is specified, all we need to do is include the `<temperature>` element, which specifies the model's temperature in Kelvin.
 
+### Objects with a varying surface temperature
+
+Most of the time, objects have a varying surface temperature.
+Here's an example of the [rescue randy](https://app.ignitionrobotics.org/OpenRobotics/fuel/models/Rescue%20Randy%20Heat%20Signature) model, which has a "heat signature" applied to it.
+The heat signature is a texture that can be applied to a model, which specifies the model's temperature across its surface.
+
+If we take a look at rescue randy's `model.sdf`, we can see that it incorporates the thermal system plugin, just like the uniform temperature box example above:
+
+```xml
+        <plugin
+          filename="ignition-gazebo-thermal-system"
+          name="ignition::gazebo::systems::Thermal">
+          <heat_signature>https://fuel.ignitionrobotics.org/1.0/openrobotics/models/rescue randy heat signature/2/files/materials/textures/RescueRandy_Thermal.png</heat_signature>
+          <min_temp>0</min_temp>
+          <max_temp>655.35</max_temp>
+        </plugin>
+```
+
+There are a few differences to note for objects with a varying surface temperature:
+* `<heat_signature>`: The path to the heat signature texture.
+This can either be an [Ignition Fuel](https://ignitionrobotics.org/libs/fuel_tools) URI, or a path to a file that is on your machine locally.
+If using paths to local files, be sure to use absolute paths.
+* `<min_temp>`: The minimum temperature that should be represented by the heat signature.
+For this example, the temperature units are in Kelvin.
+* `<max_temp>`: The maximum temperature that should be represented by the heat signature.
+655.35 is the maximum value for a 16-bit thermal camera with a 10mK linear resolution.
```

Comment:
Add units.

Reply:
Added in https://github.com/ignitionrobotics/ign-sensors/pull/79/commits/2c53bffa31eb3014499439cbdae9d5902fbd37ab